lib/providers/env_var_provider.ex

```diff
@@ -58,27 +58,40 @@ defmodule EnvVar.Provider do
   in the config files for values that are configured in the EnvVar
   Provider.
   """
-  def init(prefix: prefix, env_map: env_map, enforce: enforce) when is_atom(prefix) do
```

Comment:
With this approach you're force to use this order when passing the options. With the new approach, you're not (which is the common way to do this in Elixir).

Reply:
I'm definitely for this change. Wish there were a cleaner/nicer way to do this natively in Elixir :(